<ui version="4.0" >
 <class>GlobalProxySettingsClass</class>
 <widget class="QWidget" name="GlobalProxySettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GlobalProxySettings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QGroupBox" name="proxyBox" >
        <property name="title" >
         <string>Proxy</string>
        </property>
        <layout class="QGridLayout" name="_17" >
         <item row="0" column="0" >
          <layout class="QHBoxLayout" name="_18" >
           <item>
            <layout class="QVBoxLayout" name="_19" >
             <item>
              <widget class="QLabel" name="typeLabel_2" >
               <property name="text" >
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="proxyHostLabel_2" >
               <property name="text" >
                <string>Host:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="proxyPortLabel_2" >
               <property name="text" >
                <string>Port:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="_20" >
             <item>
              <layout class="QVBoxLayout" name="_21" >
               <item>
                <widget class="QComboBox" name="typeBox_2" >
                 <property name="minimumSize" >
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text" >
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>HTTP</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>SOCKS 5</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="proxyHostEdit_2" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="_22" >
               <item>
                <widget class="QSpinBox" name="proxyPortBox_2" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="minimum" >
                  <number>1</number>
                 </property>
                 <property name="maximum" >
                  <number>65535</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="authBox_2" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Authentication</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <layout class="QHBoxLayout" name="_23" >
           <item>
            <layout class="QVBoxLayout" name="_24" >
             <item>
              <widget class="QLabel" name="userNameLabel_2" >
               <property name="text" >
                <string>User name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="userPasswordLabel_2" >
               <property name="text" >
                <string>Password:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="_25" >
             <item>
              <widget class="QLineEdit" name="userNameEdit_2" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="userPasswordEdit_2" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0" >
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>authBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>userNameEdit_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel" >
     <x>128</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>userPasswordEdit_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>109</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
